Client/game_sa/CModelInfoSA.h

```diff
@@ -158,7 +162,7 @@ class CBaseModelInfoSAInterface
     unsigned char  ucAlpha : 8;                         // +12
 
     unsigned char  ucNumOf2DEffects : 8;            // +13
-    unsigned short usUnknown : 16;                  // +14     Something with 2d effects
+    short          s2DEffectIndex : 16;            // +14 
```

Comment:
`: 16` doesnt make much sense here

Reply:
I prefer not to touch it. I don't want to go beyond the purpose of this PR